Stripe/PublicHeaders/STPCard.h

```diff
@@ -78,6 +78,15 @@ typedef NS_ENUM(NSInteger, STPCardFundingType) {
 + (STPCardFundingType)fundingFromString:(NSString *)string;
 
 /**
+ * Returns a string representation for the provided card funding
+ *
+ * @param funding The enum value you want to covert to a string
+ *
+ * @return A string representation of the funding, suitable for displaying to a user
+ */
++ (NSString *)stringFromFunding:(STPCardFundingType)funding;
```

Comment:
nit: would prefer calling this `displayStringFromFunding`, to make clear that it's not the inverse of `fundingFromString`